Sources/MapboxNavigation/InstructionPresenter.swift

```diff
@@ -212,38 +220,43 @@ class InstructionPresenter {
     
     private func legacyAttributedString(for legacyIcon: UIImage,
                                         dataSource: DataSource) -> NSAttributedString {
+        let image = legacyIcon.scale(to: dataSource.font.pointSize * 1.2)
         let attachment = ShieldAttachment()
         attachment.font = dataSource.font
-        attachment.image = legacyIcon
+        attachment.image = image
         return NSAttributedString(attachment: attachment)
     }
 
     private func shieldAttributedString(for representation: VisualInstruction.Component.ImageRepresentation,
+                                        idiom: UIUserInterfaceIdiom,
                                         in repository: SpriteRepository,
                                         dataSource: DataSource) -> NSAttributedString? {
         guard let shield = representation.shield,
-              let cachedImage = repository.roadShieldImage(from: shield) else { return nil }
+              let cachedImage = repository.roadShieldImage(from: shield, idiom: idiom) else { return nil }
 
         let attachment = ShieldAttachment()
         attachment.font = dataSource.font
         let shieldColor = dataSource.shieldColor(from: shield.textColor)
         let fontSize = dataSource.font.with(multiplier: 0.4)
-        attachment.image = cachedImage.withCenteredText(shield.text,
-                                                        color: shieldColor,
-                                                        font: fontSize)
+        let shieldWithCenteredText = cachedImage.withCenteredText(shield.text,
+                                                                  color: shieldColor,
+                                                                  font: fontSize)
+        let image = shieldWithCenteredText.scale(to: dataSource.font.pointSize * 1.2)
```

Comment:
It'd be good to extract `1.2` into separate constant as it's called from two places.